src/seml/start.py

```diff
@@ -123,6 +125,15 @@ def get_command_from_exp(
                 f"Starting debug server with IP '{ip_address}' and port '{port}'. "
                 f'Experiment will wait for a debug client to attach.'
             )
+            previous_root = logging.root
+            try:
+                logging.basicConfig(force=True, level=logging.INFO)
```

Comment:
Why do we need the new logger?

Reply:
is this due to the long links?

Reply:
Yes; Changed it to using the rich console.